Cargo.toml

```diff
@@ -12,7 +16,7 @@ prost-types = "0.12"
 bytes = "1.5.0"
 flate2 = "1.0.28"
 hex = "0.4.3"
-starknet_api = "0.8.0"
+starknet_api = "0.7.0-dev.0"
```

Comment:
I wonder if we should drop this dependency completely 🤔 It might cause some compatibility problems further.

Reply:
We probably should